projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -0,0 +1,147 @@
+			'nonce'                    => wp_create_nonce( 'wp_rest' ),
+			'site_name'                => \get_bloginfo( 'name' ),
+			'sections'                 => array(),
+			// Features are inlined in Calypso Blaze app (wp-calypso/apps/blaze-dashboard)
+			'features'                 => array(),
+			'intial_state'             => array(
+				'currentUser' => array(
+					'id'           => 1000,
+					'user'         => array(
+						'ID'         => 1000,
+						'username'   => 'no-user',
+						'localeSlug' => $this->get_site_locale(),
+					),
+					'capabilities' => array(
+						"$blog_id" => $this->get_current_user_capabilities(),
+					),
+				),
+				'sites'       => array(
+					'items'    => array(
+						"$blog_id" => array(
+							'ID'           => $blog_id,
+							'URL'          => site_url(),
+							'jetpack'      => true,
+							'visible'      => true,
+							'capabilities' => $empty_object,
+							'products'     => array(),
+							'plan'         => $empty_object, // we need this empty object, otherwise the front end would crash on insight page.
+							'options'      => array(
+								'wordads'    => ( new Modules() )->is_active( 'wordads' ),
+								'admin_url'  => admin_url(),
+								'gmt_offset' => $this->get_gmt_offset(),
+							),
+						),
+					),
+					'features' => array( "$blog_id" => array( 'data' => $this->get_plan_features() ) ),
+				),
+			),
+		);
+	}
+
+	/**
+	 * Get the current site GMT Offset.
+	 *
+	 * @return float The current site GMT Offset by hours.
+	 */
+	protected function get_gmt_offset() {
+		return (float) get_option( 'gmt_offset' );
+	}
+
+	/**
+	 * Page base for the Calypso admin page.
+	 */
+	protected function get_admin_path() {
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		if ( ! isset( $_SERVER['PHP_SELF'] ) || ! isset( $_SERVER['QUERY_STRING'] ) ) {
+			$parsed = wp_parse_url( admin_url( 'tools.php?page=advertising' ) );
+			return $parsed['path'] . '?' . $parsed['query'];
+		}
+		// We do this because page.js requires the exactly page base to be set otherwise it will not work properly.
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		return wp_unslash( $_SERVER['PHP_SELF'] ) . '?' . wp_unslash( $_SERVER['QUERY_STRING'] );
+	}
+
+	/**
+	 * Get locale acceptable by Calypso.
+	 */
+	protected function get_site_locale() {
+		/**
+		 * In WP, locales are formatted as LANGUAGE_REGION, for example `en`, `en_US`, `es_AR`,
+		 * but Calypso expects language-region, e.g. `en-us`, `en`,  `es-ar`. So we need to convert
+		 * them to lower case and replace the underscore with a dash.
+		 */
+		$locale = strtolower( get_locale() );
+		$locale = str_replace( '_', '-', $locale );
+
+		return $locale;
+	}
+
+	/**
+	 * Get the features of the current plan.
+	 */
+	protected function get_plan_features() {
+		$plan = Current_Plan::get();
+		if ( empty( $plan['features'] ) ) {
+			return array();
+		}
+		return $plan['features'];
+	}
```

Comment:
Down the road (not in this PR), I think it may be nice to extract those instead of copying them from the Stats Admin class.